src/minimega/deploy.go

```diff
@@ -20,23 +20,29 @@ import (
 var deployCLIHandlers = []minicli.Handler{
 	{ // deploy
 		HelpShort: "copy and run minimega on remote nodes",
-		HelpLong: `deploy will copy and run minimega on remote nodes,
-facilitating the deployment of minimega on a cluster. By default, deploy will
-launch minimega with the same flags used when starting this minimega, and add
-the -nostdin flag so that the remote minimega can be backgrounded.
+		HelpLong: `
+Deploy copies and runs minimega on remote nodes, facilitating the deployment of
```

Comment:
I've been in the habit of using API names as special nouns of a sort, where the API is explicitly not capitalized. We do that explicitly for 'minimega' as well on the website, even when it's the beginning of a sentence. 

Thoughts on that practice? If it's agreeable, we should change Deploy to deploy.